docs/install_guides/pypi-openvino-dev.md

```diff
@@ -30,11 +30,10 @@ The table below lists the supported operating systems and Python* versions requi
 | :------------------------------------------------------------| :---------------------------------------------------|
 |   Ubuntu* 18.04 long-term support (LTS), 64-bit              | 3.6, 3.7                                            |
 |   Ubuntu* 20.04 long-term support (LTS), 64-bit              | 3.6, 3.7                                            |
-|   Red Hat* Enterprise Linux* 8.2, 64-bit                     | 3.6, 3.7                                            |
-|   CentOS* 7.4, 64-bit                                        | 3.6, 3.7                                            |
+|   Red Hat* Enterprise Linux* 8, 64-bit                       | 3.6, 3.7                                            |
```

Comment:
RHEL 8 only supports Python 3.6 and 3.8. It is possible to compile 3.7, but since Red Hat does not officially support it for their customers, we should not recommend it. For full details see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/using-python3_configuring-basic-system-settings